schemas/api_schema.js

```diff
@@ -185,25 +197,42 @@ type Mutation {
             editSession: async (
                 parent,
                 {
-                    user_token,
                     session_id,
                     title,
                     description,
                     category,
                     tags,
-                    start_date,
-                    end_date,
+                    time_range,
                     capacity,
                     attendees,
                     platform,
                     platform_media_id,
                     img_source
                 },
-                { db },
+                { db, user },
                 info
             ) => {
-                // TODO: sanitize the date (end > start)
-                const user_id = check_auth(user_token);
+                check_auth(user);
+
+                let start_date, end_date;
+
+                if (time_range) {
```

Comment:
you should also check that the given range is not abnormally short or long (e.g 2 seconds or 5 months)